trento/xml/sso-integration.xml

```diff
@@ -0,0 +1,762 @@
+          <literal>SAML_USERNAME_ATTR_NAME</literal>,
+          <literal>SAML_EMAIL_ATTR_NAME</literal>,
+          <literal>SAML_FIRSTNAME_ATTR_NAME</literal> and
+          <literal>SAML_LASTNAME_ATTR_NAME</literal>. So for example, if
+          the IDP user profile username is defined as
+          <literal>attr:username</literal>,
+          <literal>SAML_USERNAME_ATTR_NAME=attr:username</literal>
+          should be used.
+        </para>
+      </section>
+      <section xml:id="signing-keys">
+        <title>Signing keys</title>
+        <para>
+          Commonly, SAML protocol messages are signed with SSL. This is
+          optional using Trento, and the signing is not required (even
+          though it is recommended). If the IDP signs the messages, and
+          expect signed messages back, certificates used by the SP
+          (Trento in this case) must be provided to the IDP, the
+          Certificate file in this case.
+        </para>
+        <para>
+          For this reason, Trento already provides a certificates set
+          created during the installation. When Trento is installed the
+          first time (does not matter the installation mode) the
+          certificates are created, and the public certificate file
+          content is available in the
+          <literal>http://localhost:4000/api/public_keys</literal>
+          route.
+        </para>
+        <programlisting language="bash">
+curl http://localhost:4000/api/public_keys
+</programlisting>
+        <para>
+          Copy the content of the certificate from there, and provide it
+          to the IDP. This way, the IDP will sign and verify the
+          messages sent by both ends.
+        </para>
+        <para>
+          When this certificate is used, and provided to the IDP, the
+          IDP recreates its own <literal>metadata.xml</literal> file.
+          This file defines which certificate is used to sign the
+          messages by both sides. At this point, Trento Web must be
+          restarted to use the new <literal>metadata.xml</literal>
+          content.
+        </para>
+        <para>
+          If the <literal>SAML_METDATA_CONTENT</literal> option is being
+          used, the content of this variable must be updated with the
+          new metadata. In the other hand, if
+          <literal>SAML_METADATA_URL</literal> is used, the new metadata
+          is automatically fetched. If neither of these steps are
+          completed, communication will fail because the message
+          signatures will not be recognized
+        </para>
+        <para>
+          <emphasis role="strong">This restart must be done manually, by
+          the admin. If the installation is done by a
+          <literal>RPM</literal>, restarting the
+          <literal>systemd</literal> daemon. If the installation is done
+          using <literal>Docker</literal>, the container must be
+          restarted.</emphasis>
```

Comment:
I limit the emphasis to the core sentence, not the whole paragraph.

```suggestion
          <emphasis role="strong">This restart must be done manually, by
          the admin.</emphasis> If the installation is done by a
          <literal>RPM</literal>, restarting the
          <literal>systemd</literal> daemon. If the installation is done
          using <literal>Docker</literal>, the container must be
          restarted.</emphasis>
```

If you need a more visible appearance, use `<note>` or even `<important>`.